core/ajax.py

```diff
@@ -871,6 +871,11 @@ def import_kodi_movies(self, movies):
 
         for movie in movies:
 
+            if not movie['imdbid']:
```

Comment:
If Kodi doesn't provide an imdbid we should give the user a chance to supply it in the webui similar to how the Plex and Directory import pages handle missing data.

Reply:
Understand but do changing behaviour in ui prevents the need to control there is actually something to query tmdb for ?
You might have personal videos that do not have IMDb Id: it is not a missing metadata from Kodi search.

But it indeed it would be good to have mean to hand over the process and found missing ones.